xscertparser/cmd/acklogparser.py

```diff
@@ -268,12 +268,22 @@ def post_json_to_mongodb(json):
 
 
 def validate_test_run(json):
+    xs_version = json['global_config']['xs_version'].split('.')
+    xs_version = tuple([int(i) for i in xs_version])
+
+    # CP-30556: Deprecate Marvell (Qlogic) legacy drivers for CH 8.0
+    driver_black_list = ["qla4xxx", "qla3xxx", "netxen_nic", "qlge", "qlcnic"]
+
     for dev in json['devices']:
         print ""
         if dev['tag'] == 'NA':
             print dev['PCI_description']
             print "Driver: %s %s" % (dev['Driver'], dev['Driver_version'])
             print "Firmware: %s" % dev['Firmware_version']
+
+            if xs_version >= (8, 0, 0) and dev['Driver'] in driver_black_list:
```

Comment:
Did not realise you could use tuples like this to compare complex version numbers! Will be re-using this in the future 👍